### PR TITLE
DEVDOCS-3223-update-template-objects

### PR DESCRIPTION
**Pull request by traci707**
[DEVDOCS-3223](https://jira.bigcommerce.com/browse/DEVDOCS-3223)
Updated Handlebars Email Template Objects per [LOCAL-760](https://jira.bigcommerce.com/browse/LOCAL-760)

### CODE REVIEW

File: models/email_templates/combined_order_status_email.yml [hunk 1]
```diff
@@ -117,7 +123,14 @@ properties:
       full_name:
         type: string
       email:
```
Comment (bc-yehor-kravchenko) [inline models/email_templates/combined_order_status_email.yml:125]: No type specified for email field

File: models/email_templates/gift_certificate_email.yml [hunk 2]
```diff
@@ -71,7 +71,14 @@ properties:
       full_name:
         type: string
       email:
-        type: string
```
Comment (bc-yehor-kravchenko) [inline models/email_templates/gift_certificate_email.yml:74]: No type specified for email field

**Review by bc-tgomez — APPROVED** [review]
Compared requested changes in Jira to changes made by Traci, g2g. No suggestions made.

File: models/email_templates/invoice_email.yml [hunk 3]
```diff
@@ -205,13 +205,15 @@ properties:
...
             type: string
           value:
-            type: integer
+            type: float
       meta:
         type: object
         properties:
           mandate_url:
             type: string
             description: Link to the confirmation page in the Stripe.
```
Comment (markcmurphy) [inline models/email_templates/invoice_email.yml:214]:
```suggestion
            description: Link to the confirmation page in Stripe.

```

File: models/email_templates/invoice_email.yml [hunk 4]
```diff
@@ -205,13 +205,15 @@ properties:
...
+          mandate_tag:
```
Comment (markcmurphy) [inline models/email_templates/invoice_email.yml:215]: Should we put add a description here? It seems like this relates to Stripe direct debit transactions (https://stripe.com/docs/sources/sepa-debit) but I didn't see anything in their docs about tags. Will readers understand what this is?

File: models/email_templates/create_guest_account_email.yaml [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+          password_label:
+            type: string
+          password_placeholder:
+            type: string
+          sign_in:
+            type: string
+          help:
+            type: string
+          go_shopping:
+            type: string
```
Comment (markcmurphy) [inline models/email_templates/create_guest_account_email.yaml:95]:
```suggestion
            type: string
            
```

**Review by bc-yehor-kravchenko — APPROVED** [review]

File: models/email_templates/invoice_email.yml (hunk 4 above)
Comment (traci707) [reply models/email_templates/invoice_email.yml:215]: @markcmurphy I received the following description for `mandate_tag`. 
`mandate_tag`: Short name of the payment document
See the DEVDOCS ticket for more information.

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**